specification/compute/resource-manager/Microsoft.Compute/stable/2021-07-01/compute.json

```diff
@@ -12452,6 +12455,10 @@
         "userData": {
           "type": "string",
           "description": "UserData for the virtual machines in the scale set, which must be base-64 encoded. Customer should not pass any secrets in here. <br><br>Minimum api-version: 2021-03-01"
+        },
+        "capacityReservation": {
+          "$ref": "#/definitions/CapacityReservationProfile",
```

Comment:
CapacityReservationProfile does not exist as a schema definition yet. It needs to be added or else this reference is null.